src/components/Modal.module.scss

```diff
@@ -0,0 +1,32 @@
+@import 'src/styles/colors';
+
+.ModalOverlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background-color: rgba(0,0,0,0.7);
```

Comment:
Perfect way to achieve a semi-transparent background! Small tip: in SCSS, you can use hex values with `rgba` function, like so:
```scss
background-color: rgba(#000, 0.7);
```